Makefile

```diff
@@ -214,6 +214,20 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 		-t ../../../Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md \
 		-t ../../../Documentation/Helm-Charts/_templates.gotmpl
 
+check-helm-docs:
```

Comment:
please add this in the developer rook guide

Reply:
@parth-gr 
> please add this in the developer rook guide

checking how to do this and what exactly to add ...

Reply:
https://rook.io/docs/rook/v1.12/Contributing/development-flow/#submitting-a-pull-request somewhere here, but I think there is no make section, maybe good to start a new ection

Reply:
@parth-gr  wrote:
> please add this in the developer rook guide

done, thanks!

Reply:
> https://rook.io/docs/rook/v1.12/Contributing/development-flow/#submitting-a-pull-request somewhere here, but I think there is no make section, maybe good to start a new ection

good idea! I didn't do exactly this, but I expanded th "Running helm-docs" section https://github.com/rook/rook/blob/master/Documentation/Contributing/documentation.md#running-helm-docs

Reply:
@parth-gr, go you suggest/request to further expanding the guide by adding a section in this PR?

Reply:
Its fine